outofcore/include/pcl/outofcore/visualization/outofcore_cloud.h

```diff
@@ -51,7 +52,7 @@ class OutofcoreCloud : public Object
 //    using CloudDataCacheIterator = std::map<std::string, vtkSmartPointer<vtkPolyData> >::iterator;
 
 
-    static boost::shared_ptr<std::thread> pcd_reader_thread;
+    static std::shared_ptr<std::thread> pcd_reader_thread;
```

Comment:
This one appear to be a public api member. I'm just asking if it was intentional given what you wrote in the commit message.

Reply:
Right, this is public, though arguably shouldn't be, so I just cut corners here.